build.cake

```diff
@@ -25,7 +25,7 @@ var packageVersion = version + modifier + dbgSuffix;
 
 var PROJECT_DIR = Context.Environment.WorkingDirectory.FullPath + "/";
 var PACKAGE_DIR = PROJECT_DIR + "package/";
-var BIN_DIR = PROJECT_DIR + "bin/" + configuration + "/";
+var BIN_DIR = PROJECT_DIR + "bin/" + configuration + "/net35/";
```

Comment:
@rprouse Does CreateImage include the .NET Standard version of the engine?

Reply:
I think the test projects were upgraded to .NET 3.5 to support NSubstitute.

Reply:
It makes me a little nervous copying a dll from the `net35` directory and packaging it under `net20` - but it would take a fair bit of work to redo, and probably isn't worth the effort. 🙂